Dockerfile

```diff
@@ -52,9 +67,6 @@ RUN chmod +x /app/scripts/entrypoint.sh && \
   adduser -h /app -g keygen -u 1000 -s /bin/bash -D keygen && \
   chown -R keygen:keygen /app
 
-COPY --from=build --chown=keygen:keygen \
```

Comment:
Does it matter that `COPY --chown=keygen:keygen` is run before `RUN adduser -g keygen -D keygen` here?

Why was the `COPY` moved higher up? Just for my own edification.

Reply:
The difference is that the `COPY .` command is right before those files are needed. This makes the rebuilds faster when a file changes in the repository as Docker caches all the previous steps including this bundle COPY.

Although I missed this chown here, which is now fixed in [`57dd1f6` (#813)](https://github.com/keygen-sh/keygen-api/pull/813/commits/57dd1f638ddfc79b6d56bac2d5463cd2bf91f07a)